crates/cargo-test-support/src/compare.rs

```diff
@@ -201,6 +201,18 @@ static MIN_LITERAL_REDACTIONS: &[(&str, &str)] = &[
     ("[EXE]", std::env::consts::EXE_SUFFIX),
     ("[BROKEN_PIPE]", "Broken pipe (os error 32)"),
     ("[BROKEN_PIPE]", "The pipe is being closed. (os error 232)"),
+    // Unix message for an entity was not found
+    ("[NOT_FOUND]", "No such file or directory (os error 2)"),
+    // Windows message for an entity was not found
+    (
+        "[NOT_FOUND]",
+        "The system cannot find the file specified. (os error 2)",
+    ),
+    (
+        "[NOT_FOUND]",
+        "The system cannot find the path specified. (os error 3)",
+    ),
+    ("[NOT_FOUND]", "program not found"),
```

Comment:
Is this a case where `program not found` is shown on one platform and the other messages are shown on the others?

Reply:
Yes as reported in https://github.com/rust-lang/cargo/pull/14068/files#r1639170882 and CI https://github.com/weihanglo/cargo/actions/runs/9509316080/job/26212076581#step:11:4107.